Document prereq for ADO Universal Artifact Feed (#1308)
Azure/ResourceModules: docs/wiki/Getting started - Scenario 1 Onboard module library and CI environment.md

To use the pipelines that come with the environment in Azure DevOps, you need to register them first. You can either do this manually, or, execute the utility `Register-AzureDevOpsPipeline` we provide in path `utilities/tools/AzureDevOps`. For further information, please refer to the corresponding [documentation](./Interoperability%20-%20Register%20Azure%20DevOps%20Pipelines).

### 3.2.5 Azure Artifacts Universal Packages

This section will explain what is required to publish the modules to [Azure Artifacts Universal Packages](https://docs.microsoft.com/en-us/azure/devops/artifacts/quickstarts/universal-packages?view=azure-devops). It will also assume you are publishing from Azure DevOps Pipelines.
#### The dependent components are
1. An Azure DevOps organization

> 1. An Azure Artifacts feed titled `ResourceModules`

Reviewer: ```suggestion
1. An Azure DevOps Artifacts feed
```

> > By default, the feed name is titled `ResourceModules` under `./global.variables.yaml` under the variable name `vstsFeedName`

Reviewer: ```suggestion
> Note: The default feed name is `ResourceModules` as configured in the `./global.variables.yaml` file's variable `vstsFeedName`. Update the value hear if you want to use a different name.
```

1. An Azure DevOps project to host the artifact feed

> > Note: There are a couple options to consider when setting up an Azure Artifact feed. For example, organization-scoped feeds vs project-scoped feeds. Please see what option suits your needs by reviewing the [feeds](https://docs.microsoft.com/en-us/azure/devops/artifacts/concepts/feeds?view=azure-devops) document first.

Reviewer: ```suggestion
   > Note: There are a couple options to consider when setting up an Azure Artifact feed. For example, organization-scoped feeds vs project-scoped feeds. Please see what option suits your needs by reviewing the [feeds](https://docs.microsoft.com/en-us/azure/devops/artifacts/concepts/feeds?view=azure-devops) document first.
```
